optuna/study.py

```diff
@@ -481,6 +481,23 @@ def stop(self) -> None:
         spawned finishes.
         This method does not affect any behaviors of parallel or successive study processes.
 
+        Example:
+
+            .. testcode::
+
+                import optuna
+
+                study = optuna.create_study()
+
+                def objective(trial):
+                    if trial.number == 4:
+                        study.stop()
+                    x = trial.suggest_uniform("x", 0, 10)
+                    return x ** 2
+
+                study.optimize(objective, n_trials=10)
```

Comment:
A nit. Not really a convention but for consistency with other code examples.
```suggestion
                def objective(trial):
                    if trial.number == 4:
                        study.stop()
                    x = trial.suggest_uniform("x", 0, 10)
                    return x ** 2
                    
                study = optuna.create_study()
                study.optimize(objective, n_trials=10)
```

Reply:
Thank you for reviewing PR. I fixed it.